lib/InfluxDB/HTTP.pm

```diff
@@ -215,6 +232,8 @@ database.
 The returned object evaluates to true if the write was successful, and otherwise to
 false.
 
+The optional argument precision can be given if a precsion different than "ns" is used in the line protocol.  InfluxDB docs suggest that using a coarser precision than ns can save space and processing. In many cases "s" or "m" might do.
```

Comment:
@niclan Please add line breaks to the documentation such that each line is approximately as long as the preceding ones.